src/ja/home.html

```diff
@@ -0,0 +1,106 @@
+---
+title: 'Open Web Advocacy'
+metaDesc: 'オープンなウェブの未来を擁護するために世界中から集まったソフトウェアエンジニアのグループ'
+permalink: '/ja/'
+layout: 'layouts/home.njk'
+isHome: true
+---
+
+<p class="home-intro">
+  私たちは<strong>オープンなウェブの未来</strong>を擁護するために世界中から集まったソフトウェアエンジニアのグループです。規制当局、立法者、政策立案者に対して複雑な技術の詳細情報を提供することで、この業界における競争を阻害する主要な問題とその解決方法の理解をサポートしています。
+</p>
+
+<div class="highlight">
+  <h2>危機にさらされているのは何か？</h2>
+
+  <div class="group">
+    <p>アプリケーション開発の未来すべてが危機にさらされています。規制や立法による変化なくしては、普遍的で、無料かつオープンな、一度書くだけでよく、どこにでもデプロイ可能な、アプリケーション頒布とデプロイのシステムを失うリスクがあります。このシステムは、企業と消費者のコストを大幅に削減するシステムです。
+    <p>もし規制や立法による変化がなければ、資金はプロプライエタリなエコシステムに向かい、ゲートキーバーたちは重税を徴収することが可能になります。これにより、ゲートキーバーのコントロール下にロックインされ、今後何年にもわたってモバイルアプリのイノベーションが阻害されることになってしまいます。</p>
+  </div>
+</div>
+
+<div class="presentation">
+  <div class="policy-entry">
+    <h2>どこから始めればいい？</h2>
+    <p>開発者のアドホック・グループとして、私たちは<a href="/ja/apple-browser-ban">#AppleBrowserBan</a>（Appleによるブラウザーの禁止）がオープンなウェブの未来に対する大きな脅威であると確信しました。イギリスの独占規制当局である、競争・市場庁（Competition and Markets Authority、CMA）が<a href="https://www.gov.uk/cma-cases/mobile-ecosystems-market-study">モバイルエコシステムに対する市場調査</a>を立ち上げた際に、何人かのイギリスの「メンバー」が、CMAに対してブリーフィングを行うために招待されました。</p>
+
+    <p>私たちはこの調査の一環として、規制当局への提出書『<a href="/ja/walled-gardens-report/">Bringing Competition to Walled Gardens（ウォールド・ガーデンに競争をもたらす）</a>』を作成しました。</p>
+
+    <!-- <p><a href="/ja/policy/" class="action-button">About our actions</a></p> -->
+  </div>
+
+  <div class="priorities-entry">
+    <h2>優先すべきことトップ3</h2>
+
+    <dl class="counter">
+      <dt>
+        <a href="/ja/apple-browser-ban">#AppleBrowserBan</a>
+      </dt>
+      <dd><strong>Appleによるサードパーティ製ブラウザの禁止</strong>は、iOS上で行われてきた非常に反競争的な行為です。Safari/WebKitチームの資金援助を制限することで、過去10年間に渡ってイノベーションを停滞させ、モバイル上でウェブアプリが発展することを妨げてきました。</dd>
+
+      <dt>深いシステム統合</dt>
+      <dd><strong>ウェブアプリは普通のアプリになる必要があります。</strong>無料でオープンなウェブ技術で構築されたアプリには、平等な扱いと統合が必要です。閉鎖的で重税が課されたプロプライエタリなエコシステムは、いかなる優遇も受けるべきではありません。
+
+      <dt>ウェブアプリの平等性</dt>
+      <dd><strong>ゲートキーバーたちによって設置された人工的な障壁は、すべて取り除かなければなりません。</strong>ウェブアプリは、許可さえあれば、要求の厳しいユースケースに対して、より優れたプライバシーとセキュリティを備えた同等の機能を提供できます。</dd>
```

Comment:
- "要求の厳しい"はちょっとこっちが悪者感でてるかも。"より多くの"とかに変えたほうがいいと思う
- "ユースケース"は非カタカナ語にするのはむずかしそう
- "プライバシー"・"セキュリティ"は一般語彙になっていると思う
- "can"は[これ](https://eow.alc.co.jp/search?q=can)の7個目の意味で使ってませんか？もしそうなら最後を以下の通り変えたほうがいい
```suggestion
      <dd><strong>ゲートキーバーたちによって設置された人工的な障壁は、すべからく取り除かなければなりません。</strong>許可されたウェブアプリは、より多くのユースケースに対して、より優れたプライバシーとセキュリティを備えて同等の機能を提供できるべきです。</dd>
```